routes/index.js

```diff
@@ -34,6 +34,12 @@ routes.POSTlogin = function(req, res, next) {
   })(req, res, next);
 }
 
+routes.GETlogout = function(req, res){
+  console.log("logged out")
+  req.logout();
+  res.redirect("/")
```

Comment:
so, when you press logout, this happens:
a href -> /
call to GETlogout -> /
controller on success -> /logout
I think we should limit these to just one redirection instead of 3